integration-tests/sanity/sanity_test.go

```diff
@@ -1,77 +1,374 @@
 package sanity
 
 import (
+	"time"
+
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+
 	. "github.com/onsi/ginkgo"
-	// . "github.com/onsi/gomega"
+	. "github.com/onsi/gomega"
+	"github.com/openebs/maya/integration-tests/artifacts"
+	"github.com/openebs/maya/integration-tests/kubernetes"
+	k8s "github.com/openebs/maya/pkg/client/k8s/v1alpha1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-var _ = Describe("Sanity", func() {
-	Context("should create cstor volume", func() {
-		PIt("should apply the cstor pvc", func() {
+var (
+	cstorpvname string
+	jivapvname  string
+	present     bool
+)
 
+var _ = Describe("Sanity", func() {
+	Context("should update storageclass", func() {
```

Comment:
Avoid dependencies as much as possible
Avoid using artifacts in actual tests as much as possible
Avoid using Context(), It() etc functions to code the dependencies.
Try to use ginkgo's `BeforeXXX` `AfterXXX` functions.